### ISSUE TITLE
unused variable, false positive

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1182]
```abap
DATA lv_index1 TYPE i.
DATA lv_index2 TYPE i.
lv_index1 = lv_index2 = 2.
```

![image](https://user-images.githubusercontent.com/5888506/90336591-27663e80-dfdd-11ea-8451-2ed11d9377be.png)
